tests/api_connexion/endpoints/test_extra_link_endpoint.py

```diff
@@ -15,24 +15,198 @@
 # specific language governing permissions and limitations
 # under the License.
 import unittest
+from unittest import mock
+from urllib.parse import quote
 
-import pytest
+from parameterized import parameterized
+from test_utils.mock_plugins import mock_plugin_manager
 
+from airflow import DAG
+from airflow.models.baseoperator import BaseOperatorLink
+from airflow.models.dagrun import DagRun
+from airflow.models.xcom import XCom
+from airflow.plugins_manager import AirflowPlugin
+from airflow.providers.google.cloud.operators.bigquery import BigQueryExecuteQueryOperator
+from airflow.utils.dates import days_ago
+from airflow.utils.session import provide_session
+from airflow.utils.timezone import datetime
+from airflow.utils.types import DagRunType
 from airflow.www import app
+from tests.test_utils.db import clear_db_runs, clear_db_xcom
 
 
 class TestGetExtraLinks(unittest.TestCase):
     @classmethod
     def setUpClass(cls) -> None:
         super().setUpClass()
-        cls.app = app.create_app(testing=True)  # type:ignore
+        with mock.patch.dict('os.environ', SKIP_DAGS_PARSING='True'):
+            cls.app = app.create_app(testing=True)  # type:ignore
+
+    @provide_session
+    def setUp(self, session) -> None:
+        self.now = datetime(2020, 1, 1)
```

Comment:
```suggestion
        self.default_time = datetime(2020, 1, 1)
```
I suggest we use default_time instead of now

Reply:
Fixed. Thanks.